packages/zowe-explorer/src/extension.ts

```diff
@@ -66,6 +67,16 @@ export async function activate(context: vscode.ExtensionContext): Promise<ZoweEx
     // Determine the runtime framework to support special behavior for Theia
     globals.defineGlobals(preferencesTempPath);
 
+    if (PersistentFilters.getDirectValue("zowe.hideTempFolder") as boolean) {
```

Comment:
Adding this code to the activation means that the other has to restart the editor when they change the hideTempFolder. If you would rather do via an `onDidChangeConfiguration()` event handler then the files.excludes settings gets added as soon as the user saves the hideTempFolder settings change without a restart. I think it works something like this (also in activation):

```ts
context.subscriptions.push(
        vscode.workspace.onDidChangeConfiguration(async event => {
            if (event.affectsConfiguration("zowe.hideTempFolder") &&
                PersistentFilters.getDirectValue("zowe.hideTempFolder") as boolean) {
...
```

Reply:
I also added it to the event handler as suggested. Thanks!